eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/NavigationHeader.js

```diff
@@ -58,16 +59,16 @@ const QuestionnaireContent = styled.div`
   align-items: center;
 `;
 
-export class NavigationHeader extends React.Component {
+export class UnwrappedNavigationHeader extends React.Component {
   static propTypes = {
     canAddQuestionPage: PropTypes.bool.isRequired,
     onAddQuestionPage: PropTypes.func.isRequired,
     onAddSection: PropTypes.func.isRequired,
-    questionnaire: CustomPropTypes.questionnaire.isRequired,
     canAddCalculatedSummaryPage: PropTypes.bool.isRequired,
     onAddCalculatedSummaryPage: PropTypes.func.isRequired,
     canAddQuestionConfirmation: PropTypes.bool.isRequired,
     onAddQuestionConfirmation: PropTypes.func.isRequired,
+    match: CustomPropTypes.match,
```

Comment:
This looks required otherwise the `buildMetadataPath(match.params)` could fail?

Reply:
CustomProp types have now been globally changed so match.isRequired is in component and not CustomPropsTypes.match object